Simplenote/build.gradle

```diff
@@ -45,7 +45,10 @@ android {
     }
 
     testOptions {
-        unitTests.returnDefaultValues = true
+        unitTests {
+            includeAndroidResources = true
```

Comment:
I guess this is not necessary any more, right?

Reply:
I guess not. We just have one test and it does not needed.